<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSJSCZDemoMainWindow</class>
 <widget class="QDialog" name="KSJSCZDemoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="CapturePushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DevicesComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>218</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TrigetComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>84</y>
     <width>218</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="MirrorCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>156</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="FlipCheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>156</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveImagePushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="MessageInfoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>402</y>
     <width>248</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>186</y>
     <width>218</width>
     <height>117</height>
    </rect>
   </property>
   <property name="title">
    <string>FOV</string>
   </property>
   <widget class="QSpinBox" name="ColStartSpinBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>20</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="RowStartSpinBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>52</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ColSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>20</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="RowSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>52</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
   </widget>
   <widget class="QPushButton" name="SetFovPushButton">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>28</y>
      <width>55</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>126</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixed frame rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>272</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiframe number:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MultiFrameNumberSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>8192</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="FixFrameRateDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>124</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="LutCheckBox">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>156</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lut</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
